packs/travisci/actions/hook.yaml

```diff
@@ -0,0 +1,15 @@
+name: hook
+runner_type: run-python
+description: enable or disable hooks for git repos, defaults to false if active argument not passed
```

Comment:
I would not add a default. You can make the parameter required, but not provide a default. That way, user needs to be explicit in executing action, and cannot accidentally cause a problem.